### ISSUE TITLE
Logon Message sequence number

### ISSUE THREAD

**Issue by dominhtri1995** [quickfixgo/quickfix#285]
Hi,

How do I set RawData for header to add the password in the correct way. I did 

             msg.Header.SetString(quickfix.Tag(96),"1234567890")

However I keep getting the error Sequence Number too low. If I try to set tag 141 to True, it keeps resetting sequence number and never stops. I tried to set ResetOnLogon=N but no luck

I think it is a bug because I can easily login using quickfix/n with the exact same message I sent through quickfixgo

**Comment by cbusbey** [comment]
Where are you setting this value?  In the ToAdmin callback?  Can you paste the FIX logs where this message is sent?

**Comment by dominhtri1995** [comment]
Yes I set in ToAdmin. I found a workaround but still dont know why. I set ResetOnLogon=N  and put this block in ToAdmin

```
if(t34 == 1 && err == nil){
			msg.Header.SetBool(quickfix.Tag(141),true)  //Set reset sequence
```
		}
